runtimes/cloudformation/cfnpatcher/util.go

```diff
@@ -3,10 +3,21 @@ package cfnpatcher
 import (
 	"fmt"
 	"os"
+	"regexp"
+	"strings"
 
 	"github.com/Jeffail/gabs/v2"
 )
 
+var nonAlphanumericRegex = regexp.MustCompile(`[^a-zA-Z0-9][^\W_]?`)
+
+func getParameterName(envarName string) string {
+	res := nonAlphanumericRegex.ReplaceAllStringFunc(strings.ToLower(envarName), func(str string) string {
+		return strings.ToUpper(str[1:])
+	})
+	return res
+}
+
```

Comment:
CloudFormation parameter names must be alphanumeric. This aims to extrapolate human-friendly names from envars keys.
For example, given the envar `SO_LONG_AND_THANKS_FOR_ALL_THE_FISH`, the related parameter would be `soLongAndThanksForAllTheFish`